### PR TITLE
Design improvements

### PR DESCRIPTION
**Pull request by juliushaertl**
- Make it a bit nicer
- Show upload progress bar

image
image
image

@rullzer

### CODE REVIEW

**Comment by juliushaertl** [comment]
Also fixes https://github.com/rullzer/dropit/issues/6

File: css/drop.scss [hunk 1]
```diff
@@ -0,0 +1,110 @@
+$containerMargin: 20px;
+
+#app-content-dropit {
+	display: flex;
```
Comment (rullzer) [inline css/drop.scss]: This apparently makes the dropit page be shown everywhere

**Comment by rullzer** [comment]
Cool stuff!

**Comment by juliushaertl** [comment]
Ah looks like I stashed some changes and forgot to commit them as well :see_no_evil: I'll have a look later.

File: css/drop.scss (hunk 1 above)
Comment (juliushaertl) [reply css/drop.scss]: Fix pushed.

**Pull request merged**